manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -40,15 +40,8 @@ spec:
       version: v1beta1
   description: |
       # cluster-kube-descheduler-operator
-      An operator to run descheduler on OpenShift
```

Comment:
@ingvagabund If you guys accept my PR #96 to update the description, it'll conflict with your updates here, in case you want to revert the changes to these lines 43-51.